### DISCUSSION TITLE
win打包成功memos但无法关闭程序？

### DISCUSSION THREAD

**Discussion by Ronny200** [tw93/Pake#395] [Q&A]
想关闭退出，结果只是缩小，只能结束任务这样吗？

**Comment by tw93** [comment]
看看菜单托盘里面

**Reply by Ronny200** [reply]
感谢回复，看过了没有任何图标，是win11系统
目前在用ahk快捷打开和关闭=。=

**Reply by Tlntin** [reply]
已修复：actions打包时，windows和Linux默认会显示托盘。
至于关闭变成了最小化，这个暂时算一个新特性吧，如果很多人觉得不太好，就会改回去的。
你需要拉一下最新代码以应用该更改（如果不会可以参考下面这个图）。

![image](https://user-images.githubusercontent.com/28218658/233118845-af57b4b9-ece5-48bb-8587-94837aaa5b10.png)

**Reply by Ronny200** [reply]
感谢，马上去试试

**Reply by xyy002** [reply]
目前我也是这样 无法关闭 还有什么解决办法吗

**Reply by Tlntin** [reply]
状态栏托盘关闭。

**Reply by xyy002** [reply]
> 已修复：actions打包时，windows和Linux默认会显示托盘。
> 
> 至于关闭变成了最小化，这个暂时算一个新特性吧，如果很多人觉得不太好，就会改回去的。
> 
> 你需要拉一下最新代码以应用该更改（如果不会可以参考下面这个图）。
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/28218658/233118845-af57b4b9-ece5-48bb-8587-94837aaa5b10.png)
> 
> 

如果我需要直接关闭而不是缩小我该怎么去修改呐

**Reply by Tlntin** [reply]
![image](https://user-images.githubusercontent.com/28218658/233754911-855694aa-10e7-42a0-be7a-3f86100a692a.png)
minimize改成close应该就可以了（具体建议你在编译器里面试试，防止写错api了）

**Reply by xyy002** [reply]
> ![image](https://user-images.githubusercontent.com/28218658/233754911-855694aa-10e7-42a0-be7a-3f86100a692a.png)
> 
> minimize改成close应该就可以了（具体建议你在编译器里面试试，防止写错api了）

感谢 我试一下

**Reply by xyy002** [reply]
> ![image](https://user-images.githubusercontent.com/28218658/233754911-855694aa-10e7-42a0-be7a-3f86100a692a.png)
> 
> minimize改成close应该就可以了（具体建议你在编译器里面试试，防止写错api了）

貌似不可以 我重新打包了依然是无法关闭
